tool/grind/double_check.dart

```diff
@@ -21,7 +20,7 @@ Future<void> doubleCheckBeforeRelease() async {
     fail("GITHUB_REF $ref is different than pubspec version ${pkg.version}.");
   }
 
-  if (listEquals(pkg.version.preRelease, ["dev"])) {
+  if (const ListEquality<Object?>().equals(pkg.version.preRelease, ["dev"])) {
```

Comment:
Why doesn't `listEquals()` work here anymore?

Reply:
It's defined within `lib/src/utils.dart`, which depends on the Sass AST, so we can't import it here